history/characters/34000_murloc.txt

```diff
@@ -6,7 +6,7 @@
 	dna="fllnpdjimif"
 	culture=murloc religion=water_deities
 	martial=6 diplomacy=6 stewardship=8 intrigue=6 learning=8
-	add_trait=tough_soldier add_trait=gluttonous add_trait=ambitious add_trait=kind add_trait=humble 
+	add_trait=tough_soldier add_trait=gluttonous add_trait=ambitious add_trait=kind add_trait=humble
```

Comment:
@fivesanon you deleted a lot of white spaces. If you work in Atom, disable it.
https://flight-manual.atom.io/using-atom/sections/editing-and-deleting-text/